lib/screens/create_region_screens/interactor/viewmodels/create_region_bloc.dart

```diff
@@ -70,7 +59,7 @@ class CreateRegionBloc extends Bloc<CreateRegionEvent, CreateRegionState> {
 
   void _onRegionIdChange(OnRegionIdChange event, Emitter<CreateRegionState> emit) {
     // TODO(gguij004): Pending validation usecase.
-    if (event.regionId.isEmpty) {
+    if (event.regionId.isEmpty || event.regionId.length < 8) {
```

Comment:
the region id does not need to be more than 8 chars. 

It can be up to 8 Chars

Reply:
it needs to be 12 characters long right?